src/use-swr.ts

```diff
@@ -173,7 +173,7 @@ export const useSWRHandler = <Data = any, Error = any>(
 
           // Start the request and keep the timestamp.
           CONCURRENT_PROMISES_TS[key] = getTimestamp()
-          CONCURRENT_PROMISES[key] = fn.apply(fn, fnArgs)
+          CONCURRENT_PROMISES[key] = fn(...fnArgs)
```

Comment:
I think we want to keep apply to avoid tslib applying object spread transform onto it

Reply:
should i also change here ?
https://github.com/vercel/swr/blob/86fb5da797a15079172cb079e367586ef7cb4bdd/infinite/index.ts#L138

Reply:
Yeah I think that one is also safe to change to `apply` call